tensorflow_addons/layers/normalizations_test.py

```diff
@@ -246,6 +246,81 @@ def test_groupnorm_conv(self):
             model.fit(x=x, y=y, epochs=1)
             self.assertTrue(hasattr(model.layers[0], "gamma"))
 
+    def test_groupnorm_correctness_1d(self):
+        model = tf.keras.models.Sequential()
+        norm = GroupNormalization(input_shape=(10,), groups=2)
+        model.add(norm)
+        model.compile(loss="mse", optimizer="rmsprop")
+
+        x = np.random.normal(loc=5.0, scale=10.0, size=(1000, 10))
+        model.fit(x, x, epochs=5, verbose=0)
+        out = model.predict(x)
+        out -= tf.keras.backend.eval(norm.beta)
+        out /= tf.keras.backend.eval(norm.gamma)
+
+        self.assertAllClose(out.mean(), 0.0, atol=1e-1)
+        self.assertAllClose(out.std(), 1.0, atol=1e-1)
+
+    def test_groupnorm_2d_different_groups(self):
+        groups = [2, 1, 10]
+        for i in groups:
+            model = tf.keras.models.Sequential()
+            norm = GroupNormalization(axis=1, groups=i, input_shape=(10, 3))
+            model.add(norm)
+            # centered and variance are 5.0 and 10.0, respectively
+            model.compile(loss="mse", optimizer="rmsprop")
+            x = np.random.normal(loc=5.0, scale=10.0, size=(1000, 10, 3))
+            model.fit(x, x, epochs=5, verbose=0)
+            out = model.predict(x)
+            out -= np.reshape(self.evaluate(norm.beta), (1, 10, 1))
+            out /= np.reshape(self.evaluate(norm.gamma), (1, 10, 1))
+
+            self.assertAllClose(
+                out.mean(axis=(0, 1), dtype=np.float32), (0.0, 0.0, 0.0), atol=1.1e-1
```

Comment:
`1,1e-1` seems oddly specific, any reason for that? Just curious. Does it fail for `1e-1`?

Reply:
Thanks @Squadrick.